ui_test/README.md

```diff
@@ -0,0 +1,20 @@
+A smaller version of compiletest-rs
+
+## Supported magic comment annotations
+
+* `// ignore-XXX` avoids running the test on targets whose triple contains `XXX`
+    * `XXX` can also be one of `64bit`, `32bit` or `16bit`
+* `// only-XXX` avoids running the test on targets whose triple **does not** contain `XXX`
+    * `XXX` can also be one of `64bit`, `32bit` or `16bit`
+* `// stderr-per-bitwidth` produces one stderr file per bitwidth, as they max differ significantly sometimes
+* `// error-pattern: XXX` make sure the stderr output contains `XXX`
+* `//~ ERROR: XXX` make sure the stderr output contains `XXX` for an error in the line where this comment is written
```

Comment:
So this has actually been implemented now? :D
If not, please mention that here.